swarm/network/stream/delivery.go

```diff
@@ -229,6 +230,11 @@ R:
 	for req := range d.receiveC {
 		processReceivedChunksCount.Inc(1)
 
+		if len(req.SData) > cp.DefaultSize+8 {
```

Comment:
@janos I realize that this is duplicated as per every `Validator` for every chunk type (content addressed, and MRU), however I'd be nice to know if we are receiving big chunks over the network, or we are creating them internally - therefore I leave this short-circuit here.

Reply:
👍